KomaMRIBase/src/datatypes/sequence/RF.jl

```diff
@@ -97,10 +97,13 @@ getproperty(x::Matrix{RF}, f::Symbol) = begin
 	end
 end
 
-# RF comparison
+"""
+For comparing two `RF`s custom types
+"""
 Base.isapprox(rf1::RF, rf2::RF) = begin
-    return all(length(getfield(rf1, k)) == length(getfield(rf2, k)) for k ∈ fieldnames(RF))
-        all(≈(getfield(rf1, k), getfield(rf2, k), atol=1e-9) for k ∈ fieldnames(RF))
+    return all(length(getfield(rf1, k)) == length(getfield(rf2, k)) for k ∈ fieldnames(RF)) &&
+        all(size(getfield(rf1, k)) == size(getfield(rf2, k)) for k ∈ fieldnames(RF)) &&
+        all(all(isapprox.(getfield(rf1, k), getfield(rf2, k), atol=1e-9)) for k ∈ fieldnames(RF))
```

Comment:
Not all of the functions need to be in one line, separate to improve readability.